### PR TITLE
Merge List to Development

### PR DESCRIPTION
**Pull request by fedgut**
- Created Project Model, Views and Controller
- Created Todos Model and Views

### CODE REVIEW

File: src/views/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  <div class="row" id="todos-div" >
+
+  <!--TODOS render the cards for EACH todo in project with id XXXX -->
+  </div>
+  </div>
+  </div>
+  </div>`;
+
+  mainDiv.innerHTML = initialContent;
+};
```
Comment (maelfosso) [inline src/views/index.js]: That HTML code is not clear, could indent it, please?

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello @fedgut @alissonperes 
First of all, thank you for that simple UI and UX.
However, there are some bugs to fix.

1. There is that error when opening the app 
Screenshot from 2020-03-30 18-23-20

2. The TODO creation form doesn't have a field for selecting the Project 

3. After creating a TODO, you don't reset the form or empty all the field because when opening back again for a new TODO, it comes with the previous values.

4. I can't affect a todo to a project

5. Please, I suggest you re-read this [article](http://aspiringcraftsman.com/2011/12/08/solid-javascript-single-responsibility-principle/) because you can't have a ProjectController to handle also TODO 

Kindly make the suggested changes and submit a new code review request
Thanks

Mael FOSSO

File: src/views/index.js (hunk 1 above)
Comment (alissonperes) [reply src/views/index.js]: Idented the html code as requested.

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello,
Thank you for the improvements you've done.

However, I can't add a TODO without an error directly after creating a project.
Take a look on that screenshot
Screenshot from 2020-03-31 18-40-38
You could notice there is not TODO listed but the localstorage contains the TODO.

Kindly fix that bug and submit a new code review request.
Thanks

Mael FOSSO

File: src/views/index.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                </div>
+                <div class="modal-body">
+                  <form id="create-todo-form">
+                    <div class="form-group">
+                      <label>Name</label>
+                      <input
+                        type="text"
+                        class="form-control"
+                        id="exampleInputEmail1"
+                        aria-describedby="emailHelp"
```
Comment (bellom) [inline src/views/index.js:65]: **Kindly do this before merging.....**

_**User shouldn't be able to create empty todo right?**_

You can apply `required` to all the input here
screenshot from your app below
image

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/views/index.js (hunk 2 above)
Comment (alissonperes) [reply src/views/index.js:65]: Validation for create and edit form of todos

**Pull request merged**